tests/test_templates.py

```diff
@@ -0,0 +1,157 @@
+import logging
+
+import numpy as np
+
+from pinttestdata import datadir
+from pint.templates import lcfitters, lcprimitives, lctemplate
+
+
+def gauss(x, x0, s):
+    return 1.0 / s / (2 * np.pi) ** 0.5 * np.exp(-0.5 * (x - x0) ** 2 / s ** 2)
+
+
+def test_prim_gauss_definition():
```

Comment:
Have you considered using von Mises functions rather than wrapped Gaussians? They're similar, approximately equal if narrow, and the von Mises distribution satisfies a similar maximum entropy property to the Gaussian. (Also pdf and cdf are implemented in scipy.stats.vonmises.)

Reply:
Yes, it's implemented.